package.json

```diff
@@ -17,7 +17,8 @@
 		"@changesets/cli": "^2.27.7"
 	},
 	"engines": {
-		"pnpm": ">=9"
+		"pnpm": ">=9",
+		"node": ">=22"
 	},
```

Comment:
Does it really need to be > 22. Please consider that most providers are just catching up to supporting 22 in GA. We should test with 20.

Reply:
this is the monorepo package.json so this constraint applies only to developing in the monorepo and not using the gateway